src/account/NexusSmartAccount.ts

```diff
@@ -64,7 +65,7 @@ import {
   convertSigner
 } from "./index.js"
 import {
-  GENERIC_FALLBACK_SELECTOR,
+  GENERIC_FALLBACK_SELECTOR_SELECTOR,
```

Comment:
should not be importing this in nexus. it was just a selector for a mock fallback.
@VGabriel45